### PR TITLE
Rebranding docs

### PR DESCRIPTION
**Pull request by ShihabMehboob**
Changed 'Bluemix' to 'IBM Cloud’.

### CODE REVIEW

File: en/starter/generator/command_line_tools_for_linux.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ generator and its prerequisites are installed correctly.
 >
 > * [IBM API Connect toolkit CLI](https://www.ibm.com/support/knowledgecenter/en/SSMNED_5.0.0/com.ibm.apic.toolkit.doc/capic_swift_overview.html) via `apic swiftserver`  
 > (uses v1.0.4 of the generator).
-> * [Bluemix CLI Developer Plug-in](https://console.ng.bluemix.net/docs/cloudnative/tutorial_web.html#tutorial/) via `bx dev create`  
+> * [IBM Cloud CLI Developer Plug-in](https://console.ng.bluemix.net/docs/cloudnative/tutorial_web.html#tutorial/) via `bx dev create`
```
Comment (helenmasters) [inline en/starter/generator/command_line_tools_for_linux.md]: According to the this page https://console.bluemix.net/docs/cloudnative/dev_cli.html#developercli we are talking about the IBM Cloud Developer Tools CLI (formerly referred to as the Bluemix CLI Developer Plug-in), so we should match their naming here.

File: en/starter/generator/project_layout_reference.md [hunk 2]
```diff
@@ -177,12 +177,12 @@ files will be included in the scaffolded project:
 │       └── Services/
 │            └── ServiceMongodb.swift - Swift connector to an IBM Cloud Service
 ├── manifest.yml                      - CloudFoundry deployment manifest file
-├── cli-config.yml                    - Bluemix CLI Dev plugin configuration file
+├── cli-config.yml                    - IBM Cloud CLI Dev plugin configuration file
```
Comment (helenmasters) [inline en/starter/generator/project_layout_reference.md]: 'IBM Cloud Dev Tools CLI configuration file' would better match the new naming convention.

File: en/starter/generator/command_line_tools_for_linux.md (hunk 1 above)
Comment (ShihabMehboob) [reply en/starter/generator/command_line_tools_for_linux.md]: Noted, I'll open a new PR with the changes.

File: en/starter/generator/project_layout_reference.md (hunk 2 above)
Comment (ShihabMehboob) [reply en/starter/generator/project_layout_reference.md]: Noted, I'll open a new PR with the changes.

**Pull request merged**